src/vs/workbench/contrib/debug/browser/media/debugHover.css

```diff
@@ -32,6 +32,17 @@
 	border-bottom: 1px solid rgba(128, 128, 128, 0.35);
 }
 
+.tip {
```

Comment:
I would just add the `.tip` class to the same rules from the `.title` class here so we're not duplicating values

https://github.com/microsoft/vscode/blob/7a01f2077dee045d51760a60bad78213d1f88ac3/src/vs/workbench/contrib/debug/browser/media/debugHover.css#L22